src/components/DateRangePicker.jsx

```diff
@@ -637,7 +645,7 @@ class DateRangePicker extends React.PureComponent {
           </OutsideClickHandler>
         )}
         {enableOutsideClick || input}
-      </div>
+      </div>,
```

Comment:
an alternative to the trailing comma is `((` and `))`

Reply:
@ljharb I think this just came w/ prettier? :D